pennylane/transforms/core/transform.py

```diff
@@ -19,6 +19,8 @@
 
 from .transform_dispatcher import TransformDispatcher, TransformError
 
+# pylint: disable=too-many-arguments
```

Comment:
Mind moving this closer to the relevant function instead of applying to the whole file?